### DISCUSSION TITLE
aspect-ratio CSS property works in Chromium but not in qutebrowser?

### DISCUSSION THREAD

**Discussion by Aster89** [qutebrowser/qutebrowser#6649] [Q&A] [answered]
Is it normal that the `aspect-ratio` CSS property doesn't work? If so, when will we see it working, considering that it works in Chromium? [Here's the CSS+HTML snippet](https://codepen.io/aster89/pen/BaZBZqL).

**Comment by The-Compiler** [comment, marked as answer]
According to [caniuse](https://caniuse.com/mdn-css_properties_aspect-ratio), support for the `aspect-ratio` property was added in Chromium 88. However, QtWebEngine 5.15 is based on Chromium 87 (with security backports from newer versions).

It will work with QtWebEngine 6.2 which is based on Chromium 90 and currently in beta (but not supported by PyQt and/or qutebrowser yet). Starting with Chromium 84 (i.e. with QtWebEngine 5.15.3), you can add `enable-experimental-web-platform-features` to `qt.args` (though that will also enable other features which were marked at experimental at that time).
